Tasks/DeployAzureResourceGroup/task.json

```diff
@@ -100,6 +100,35 @@
             "groupName": "advanced",
```

Comment:
Please get the json validated from a PM. The Name which appears, the help strings etc..